org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/references/settings/XMLReferenceExpression.java

```diff
@@ -65,16 +66,40 @@ public void setPrefix(String prefix) {
 	}
 
 	public boolean matchFrom(final Node node) {
+		return getFromMatcher().match(node);

```

Comment:
Please add Javadoc for this and `matchTo()`

Reply:
fixed